app/assets/stylesheets/content/_select2.scss

```diff
@@ -126,3 +115,78 @@
     margin-left: 0;
   }
 }
+
+//@todo for select2 visibility in livingStyleGuide, need another solution  it will rewrite globally
+.livingstyleguide--example {
```

Comment:
@vcostin please move livingstyleguide rules to the relevant section of `app/assets/stylesheets/styleguide.html.lsg`